Sources/OpenFeature/Provider/OpenFeatureNoOpProvider.swift

```diff
@@ -24,11 +24,11 @@ public struct OpenFeatureNoOpProvider: OpenFeatureProvider, CustomStringConverti
         try await gracefulShutdown()
     }
 
-    public func resolution(
+    public func resolution<Value: OpenFeatureValue>(
```

Comment:
TIL that we can satisfy multiple non-generic function requirements of the `Provider` protocol with a single generic function 👌

Reply:
Nice!